poppy/tests/test_zernike.py

```diff
@@ -211,7 +211,7 @@ def test_opd_expand(npix=512, input_coefficients=(0.1, 0.2, 0.3, 0.4, 0.5)):
         basis[idx] *= coeff
 
     opd = basis.sum(axis=0)
-    recovered_coeffs = zernike.opd_expand(opd, nterms=len(input_coefficients))
+    recovered_coeffs = zernike.decompose_opd(opd, nterms=len(input_coefficients))
```

Comment:
I'd probably change the name of the test function from `test_opd_expand()` to `test_decompose_opd()` (a few lines up) for continuity